src/integration-test/java/org/radarcns/redcap/integration/IntegratorTest.java

```diff
@@ -3,21 +3,16 @@
 import org.junit.Test;
 import org.radarcns.redcap.util.RedCapTrigger;
 
-import static org.radarcns.redcap.util.IntegrationUtils.TRIGGER_BODY;
-import static org.radarcns.redcap.util.IntegrationUtils.httpClient;
-import static org.radarcns.redcap.util.IntegrationUtils.mpClient;
+import static org.junit.Assert.assertEquals;
+import static org.radarcns.redcap.util.IntegrationUtils.*;
 
 public class IntegratorTest {
 
-    @Test(expected = IllegalStateException.class)
+    @Test
     public void getInputTest() {
         RedCapTrigger trigger = new RedCapTrigger(TRIGGER_BODY);
-        Integrator integrator = new Integrator(trigger, mpClient, httpClient);
+        Integrator integrator = new Integrator(trigger, mpClient);
 
-        /** This should throw exception as subject already exists in management portal
-         * as the {@link org.radarcns.redcap.EntryPointTest} has run already. The order is
-         * specifies by {@link org.radarcns.redcap.testsuite.OrderedTestSuite}.
-         **/
-        integrator.getInput();
+        assertEquals(integrator.handleDataEntryTrigger(), true);
```

Comment:
This will always be false since the redcap cannot be updated as there is no instance. Nonetheless, this has already been fixed in #34, hence you can merge without resolving the Travis build for this test.

Reply:
Oh okay I see. Thanks.